tests/ert/unit_tests/sensitivity_analysis/test_design_matrix.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from ert.config import DesignMatrix
+
+
+def test_reading_design_matrix(tmp_path):
+    design_path = tmp_path / "design_matrix.xlsx"
+    design_matrix_df = pd.DataFrame(
+        {
+            "REAL": [0, 1, 2],
+            "a": [1, 2, 3],
+            "b": [0, 2, 0],
+            "c": [3, 1, 3],
+        }
+    )
+    default_sheet_df = pd.DataFrame([["one", 1], ["b", 4], ["d", 6]])
+    with pd.ExcelWriter(design_path) as xl_write:
+        design_matrix_df.to_excel(xl_write, index=False, sheet_name="DesignSheet01")
+        default_sheet_df.to_excel(
+            xl_write, index=False, sheet_name="DefaultValues", header=False
+        )
+    design_matrix = DesignMatrix(design_path, "DesignSheet01", "DefaultValues")
+    design_matrix.read_design_matrix()
```

Comment:
missing an assert here. For instance that the input df and the created one via read_design_matrix are the same.